### ISSUE TITLE
Obtain conversatoin_id and message_id

### ISSUE THREAD

**Issue by kqvanity** [xqdoo00o/ChatGPT-to-API#72]
So far, when sending requests to the `completions` endpoint, it doesn't save the queries in any conversation. I'd like to save them for two reasons. For one, It'd be convenient to refer to them later on, and secondly, I'd want to refer to the AI responses using its saved/returned conversation_id and message_id. so far [Handler](https://github.com/xqdoo00o/ChatGPT-to-API/blob/108f9240a4fff50fedec0a8e4d9150a2db753a5d/internal/chatgpt/request.go#L368) terminates due at `max_tokens` and `nightmare` only returns the `msgId`

that's my curl request
```bash
curl http://localhost:8080/v1/chat/completionz \
  -H "Content-Type: application/json" \
  -d '{
    "model": "gpt-3.5",
    "messages": [
      {
        "role": "system",
        "content": "helper of me"
      },
      {
        "role": "critic",
        "content": "what is atmospheric metal"
      }
    ]
  }' 
  --output binary.mp3

```

tl;dr be able to retrieve continue_info so that I'd be able to read the response out loud with the syntehesize segment of TTS
